docs/requirements.txt

```diff
@@ -11,3 +11,4 @@ pymdown-extensions==10.14.3
 # mkdocstrings-python will format the docstrings using ruff
 # https://mkdocstrings.github.io/python/usage/configuration/signatures/#line_length
 ruff==0.11.2
+jsonschema2md==1.5.1
```

Comment:
Needs a trailing newline (please configure your editor to respect our `.editorconfig` file).

Please retain the alphabetical order of dependencies.